src/formio.component.ts

```diff
@@ -52,7 +52,10 @@ export class FormioComponent implements OnInit {
         if (this.form) {
             this.ready.next(true);
         }
-        else if (this.src && !this.service) {
+        else if (this.src) {
+          if (!this.service){
+            this.service = new FormioService(this.src);
+            }
             this.service = new FormioService(this.src);
```

Comment:
Sorry i forgot to remove this line.
Actually the point is if the custom service is not provided ,this goes to the  default formioservice ,
otherwise this will use our extended service 
I have fixed this issue [ #59](https://github.com/formio/ng2-formio/pull/58)**(refer component and Extended service)**  This is the way i have instantiated custom service.